### PR TITLE
destroyallsoftware

### PR DESCRIPTION
**Pull request by cocreature**
changelog_begin
changelog_end

### Pull Request Checklist

- [ ] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/master/CONTRIBUTING.md)
- [ ] Include appropriate tests
- [ ] Set a descriptive title and thorough description
- [ ] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: build.sh [hunk 1]
```diff
@@ -6,70 +6,4 @@ set -euo pipefail
...
-# Disabled on darwin since it sometimes seem to hang and this only
-# tests our dev setup rather than our code so issues are not critical.
-if [[ "$(uname)" != "Darwin" ]]; then
-  da-ghci --data yes //compiler/damlc:damlc -e ':main --help'
-fi
-
-# Check that our IDE works on our codebase
-ghcide compiler/damlc/exe/Main.hs 2>&1 | tee ide-log
-grep -q "1 file worked, 0 files failed" ide-log
+kill $(lsof -ti tcp:6865)
```
Comment (SamirTalwar-DA) [inline build.sh:9]: If this works, is it worth running it on CI (on Linux and macOS) before every build?

Comment (cocreature) [reply build.sh:9]: Yeah but I want to know if it works before :slightly_smiling_face:

Comment (SamirTalwar-DA) [reply build.sh:9]: Of course!

**Comment by cocreature** [comment]
Didn’t succeed at destroying all software so closing.

**Pull request closed without merging**